### PR TITLE
td: Add support for ${secret:...} syntax in result_url option

### PR DESCRIPTION
**Pull request by frsyuki**
This lets users to set password without writing it in plain text.

### CODE REVIEW

**Comment by frsyuki** [comment]
@komamitsu PTAL

File: digdag-tests/src/test/resources/acceptance/td/td/td_result_url.dig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+timezone: UTC
+
++run:
+  td>: query.sql
+  result_url: td://${secret:td.apikey}@/${td.database}/result_url
```
Comment (komamitsu) [inline digdag-tests/src/test/resources/acceptance/td/td/td_result_url.dig]: If the placeholder isn't converted at all or is replaced with empty string, this job should fail. So this test can check if the changes work well.
👍

**Comment by komamitsu** [comment]
👍

**Comment by frsyuki** [comment]
👍

**Pull request merged**